samples/manager/manager.js

```diff
@@ -283,31 +273,33 @@ const createRsaDevice = async (
     // optional auth parameters.
   });
 
-  const regPath = iotClient.registryPath(projectId, cloudRegion, registryId);
-  const device = {
-    id: deviceId,
-    credentials: [
-      {
-        publicKey: {
-          format: 'RSA_X509_PEM',
-          key: fs.readFileSync(rsaCertificateFile).toString(),
+  async function createDevice() {
+    // Construct request
+    const regPath = iotClient.registryPath(projectId, cloudRegion, registryId);
+    const readFileAsync = promisify(fs.readFile);
```

Comment:
This isn't necessary now that we're moving to Node 10 as the suppported Node.js version in our libraries.

You can do `const {readFile} = require('fs').promises`

Reply:
okay I will change that then

Reply:
it causing a lint error, Is there a way around it?
```
##[error]  17:20  error  The 'fs.promises' is not supported until Node.js 11.14.0 (backported: ^10.17.0). The configured version range is '>=8.0.0'  node/no-unsupported-features/node-builtins
```